homeassistant/components/conversation/strings.json

```
@@ -18,6 +18,20 @@
           "description": "Conversation agent to process your request. The conversation agent is the brains of your assistant. It processes the incoming text commands."
         }
       }
+    },
+    "reload": {
+      "name": "[%key:common::action::reload%]",
+      "description": "Reloads the intent configuration.",
+      "fields": {
+        "language": {
+          "name": "Language",
```

You can use references here.


```suggestion
          "name": "[%key:components::conversation::services::process::language::name%]",
```
Out the top of my head